python/lsst/ip/isr/isrFunctions.py

```diff
@@ -1222,3 +1224,65 @@ def countMaskedPixels(maskedIm, maskPlane):
     maskBit = maskedIm.mask.getPlaneBitMask(maskPlane)
     nPix = numpy.where(numpy.bitwise_and(maskedIm.mask.array, maskBit))[0].flatten().size
     return nPix
+
+
+def getExposureGains(exposure):
+    """Get the per-amplifier gains used for this exposure.
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`
+        The exposure to find gains for.
+
+    Returns
+    -------
+    gains : `dict` [`str` `float`]
+        Dictionary of gain values, keyed by amplifier name.
```

Comment:
Please add a note to the doc string that it will return an empty dict if the detector is None.